config/routes.rb

```diff
@@ -83,5 +87,15 @@
     end
   end
 
-  root to: 'messageboards#index'
+  root_path =
+    redirect do |_, request|
+      case Thredded.landing_page
+      when :active_topics
+        "#{request.script_name}/topics/active"
+      else
+        "#{request.script_name}/messageboards"
+      end
+    end
+
+  root to: root_path
```

Comment:
Can we make it so that the root path is always empty, and only the non-default page gets a path?

That is:

| landing_page | active topics page | messageboards page |
|-|-|-|
| `:active_topics` | `/` | `/messageboards` |
| `:messageboards` | `/topics/active` | `/` |

This will avoid the redirection and result in nicer-looking (shorter) URLs.